.github/workflows/build.yml

```diff
@@ -84,7 +84,7 @@ jobs:
         # list of Docker images to use as base name for tags
         images: |
           ekspand/trusty
-          ghcr.io/${{ github.repository_owner }}/trusty
+        # ghcr.io/${{ github.repository_owner }}/trusty
```

Comment:
Do we really need to keep all these comments? github already does a good job keeping the history of all commits so we can always go back and check instead of cluttering the code.